continuous-delivery/readme.md

```diff
@@ -0,0 +1,145 @@
+# Continuous Delivery
+
+The inspiration behind continuous delivery is constantly delivering valuable software to users and developers more frequently. Applying the principles and practices laid out in this readme will help you reduce risk, eliminate manual operations and increase quality and confidence.
+
+Deploying software involves the following principles:
+
+1. Provision and manage the cloud environment runtime for your application (cloud resources, infrastructure, hardware, services, etc).
+
```

Comment:
Would it be ok to drop the blank lines between bullet points to be more consistent with other numbered lists in the repo?